### ISSUE TITLE
RolloutSwizzlerDynamic.include wrong #import

### ISSUE THREAD

**Issue by crazyjooe** [rollout/rollout.io-ios#30]
With 0.7 and 0.8 versions, we can't compile our project, because of includes in RolloutSwizzlerDynamic.include.
First of all, it included pretty random files, both from pods and our project.
Second, some imports are in wrong format - e.g. we're using iSoma pod, and the include looked like this:
```objective-c
#include "SOMAUserSettings.h"
```
But it should be like this:
```objective-c
#import <iSoma/SOMAUserSettings.h>
```

But the main question is, of course, why these random files are even included.

**Comment by ekeren** [comment]
@crazyjooe - thanks for this info.
I think this issue should only be seen in 0.8, can you please check what is your is the version in the comments on `Rollout.h`?

**Comment by crazyjooe** [comment]
Yes, it was the case for 0.8. 0.7 caused some other errors that prevented us from using it instead.

**Comment by ekeren** [comment]
We are working on this issue on 0.8, I'll update here once it is ready. 
Can you describe what happens in 0.7.2 ?

**Comment by crazyjooe** [comment]
I tried to reproduce it this morning, but failed - 0.7.2 works for us right now.
If we remember correctly, we had toolchain-related errors, but I can't reproduce them any more.
My guess is that build was messed because of trying different pod versions without proper cleaning. 
Will report if I will succeed in reproducing, but we're planning to release with 0.7.2 this time.

**Comment by ekeren** [comment]
ok, thanks for the info. 
Will reproduce the issue with iSoma pods and verify everything works in 0.8.1 (once it is released). Will update ticker then

**Comment by crazyjooe** [comment]
Fixed in 0.8.1. Thanks.
